lib/src/collection/multimap.dart

```diff
@@ -382,8 +381,9 @@ class _WrappedMap<K, V, C extends Iterable<V>> implements Map<K, C> {
   @override
   void removeWhere(bool test(K key, C value)) {
     var keysToRemove = <K>[];
+    // TODO(cbracken): iterate over entries.
     for (final key in keys) {
-      if (test(key, this[key])) keysToRemove.add(key);
+      if (test(key, this[key]!)) keysToRemove.add(key);
```

Comment:
You only want a null check if `C` is not nullable; so

`this[key]!` --> `this[key] as C`

Reply:
Good call -- thanks! Done.